egs/swbd/s5c/local/run_durmodel.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+[ -f ./path.sh ] && . ./path.sh; # source the path.
+. cmd.sh
+. parse_options.sh || exit 1;
+
+system=exp/tri4
+durmodel_dir=$system/durmodel
+
+steps/dur_model/train.sh --cmd "$decode_cmd" \
+                         --max-duration 60 \
+                         --left-context 5 --right-context 2 \
+                         data/lang/phones $system $durmodel_dir || exit 1;
+
+(
```

Comment:
Actually I meant to put the second rescoring job (i.e. decode_eval2000_sw1_fsh_fg ) in a parallel shell. I will fix this.